Lesson16/bilge/script.js

```diff
@@ -0,0 +1,305 @@
+    return 'We can offer you the job at a lower salary.'
+  }
+  else {
+    return 'We can offer you the job at the requested salary.'
+  }
+}
+
+console.log('Ex. 11:');
+
+console.log(decideTheSalaryOnExperience(25.000, 4));
+console.log(decideTheSalaryOnExperience(45.000, 10));
+console.log(decideTheSalaryOnExperience(50.000, 13));
+
+// 12. Write a function that asks the user to enter their score on a test. If the score is less than 60, print "You failed the test". If the score is between 60 and 70 (inclusive), print "You passed the test, but your grade is not great". If the score is between 71 and 80 (inclusive), print "You got a B". If the score is between 81 and 90 (inclusive), print "You got an A". If the score is greater than 90, print "You got an A+ - Great job!".
+
+function checkGrade (score) {
+	if (score < 60) {
+    return 'You failed the test.'
+  }
+  else if (score >= 60 && score <= 70) {
+    return 'You passed the test, but your grade is not great.'
+  }
+  else if (score >= 71 && score <= 80) {
+    return 'You got a B.'
+  }
+  else if (score >= 81 && score <= 90) {
+    return 'You got a A.'
+  }
+  else {
+    return 'You got an A+ - Great job!'
+  }
+}
+
+console.log('Ex. 12:');
+
+console.log(checkGrade(59));
+console.log(checkGrade(60));
+console.log(checkGrade(80));
+console.log(checkGrade(90));
+console.log(checkGrade(91));
+
+// 13. Write a function that asks the user to enter a month number (1-12) and displays the corresponding season name. Use a switch statement to handle the different cases.
+
+function findSeason(month) {
+	
+	switch (month) {
+		case 12:
+		case 1:
+		case 2:
+				return "Winter";
+		case 3:
+		case 4:
+		case 5:
+				return "Spring";
+		case 6:
+		case 7:
+		case 8:
+				return "Summer";
+		case 9:
+		case 10:
+		case 11:
+				return "Autumn";
+		default:
+				return "Invalid month";
+	}
+}
+
+console.log('Ex. 13:');
+
+console.log(findSeason(1));
+console.log(findSeason(4));
+console.log(findSeason(13));
+
+// 14. Write a function that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or failing (D, F). Use a switch statement to handle the different cases.
+
+function checkIfTheGradeIsPassing(grade) {
+  switch (grade) {
+    case 'A':
+    case 'B':
+    case 'C':
+      return "The grade is passing.";
+    case 'D':
+    case 'F':
+      return "The grade is failing.";
+    default:
+      return "Invalid grade";
+  }
+}
+
+console.log('Ex. 14:');
+
+console.log(checkIfTheGradeIsPassing('A'));
+console.log(checkIfTheGradeIsPassing('F'));
+
+// 15. Write a function that asks the user to enter their age, and then calculates and displays their age in dog years (where one human year is equivalent to seven dog years). If the user enters a negative number or zero, display an error message.
+
+function findAgeInDogYears (dogYears) {
+  const humanYears = 7 * dogYears;
```

Comment:
Before making a calculation, check that dogYears is a positive number:

```suggestion
  if (dogYears <= 0) return 'Invalid input. Dog age should be specified as a positive number.'
  const humanYears = 7 * dogYears;
```